module/canvas/token.mjs

```diff
@@ -43,6 +51,14 @@ export default class TokenEd extends Token {
     return undefined;
   }
 
+  /**
+   * @description             Draw a karma bar for the Token representing a rate of karma points.
+   * @param {Number} number   Number of the bar being drawn
```

Comment:
In the upper doc it was "index" of the bar... Should be consistent if possible :)